docs/pywbemclicmdshelp.rst

```diff
@@ -925,18 +923,22 @@ The following defines the help output for the `pywbemcli connection select --hel
       connection. If the NAME argument is omitted, prompt for selecting one of
       the connection definitions in the connections file.
 
```

Comment:
The previous paragraph (starting on line 922) already describes this. The new paragraph has no new information and can be removed.